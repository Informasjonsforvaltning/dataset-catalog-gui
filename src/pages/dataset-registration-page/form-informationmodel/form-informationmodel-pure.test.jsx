import React from 'react';
import { shallow } from 'enzyme';
import { FormInformationModelPure } from './form-informationmodel-pure.component';

let defaultProps;
let wrapper;

beforeEach(() => {
  defaultProps = {};
  wrapper = shallow(<FormInformationModelPure {...defaultProps} />);
});

test('should render FormInformationModelPure correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
