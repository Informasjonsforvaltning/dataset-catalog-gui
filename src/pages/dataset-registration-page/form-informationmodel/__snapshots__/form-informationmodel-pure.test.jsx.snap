// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FormInformationModelPure correctly 1`] = `
<form>
  <div
    className="form-group"
  >
    <div
      className="mt-4"
    >
      <div
        className="form-group"
      >
        <withStateHandlers(Component)
          term="Dataset_informationModel"
          title="Informasjonsmodell"
        />
        <FieldArray
          component={[Function]}
          isReadOnly={false}
          languages={Array []}
          linkLabel="Lenke"
          name="informationModel"
          titleLabel="Tittel"
        />
      </div>
    </div>
  </div>
</form>
`;
